ush/create_ozinfo.sh

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
+# create global_ozinfo file for a given date
+date=$1
```

Comment:
```suggestion
date_in=$1
```
Please use a different name for this variable so that it does not conflict with the linux utility `date`